needrestart

```diff
@@ -103,6 +103,7 @@ Usage:
     -f <fe>	override debconf frontend (DEBIAN_FRONTEND, debconf(7))
     -t <seconds> tolerate interpreter process start times within this value
     -u <ui>     use preferred UI package (-u ? shows available packages)
+    -x		exit GUI when no processes are selected by default
```

Comment:
needrestart does not have a GUI, but it uses debconf if available which might present a TUI, GUI, …

Suggestion: `do not ask for restarts if no process is selected by default`